### ISSUE TITLE
Retrieve aggregate demographic information for studies 

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#793]
# TL;DR

Make it possible for researchers to request a download of overall data rather than response-specific data, especially for fields that raise reidentification risk.  E.g. "There were 12 M, 11 F, 1 undisclosed gender" when response-specific gender isn't needed for analysis

# Narrative
As a researcher, I want to report summary statistics about my sample but don't need specific data, and I want to lower reidentification risk

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

</p>
</details>
